stubs/xtea/xtea/counter.pyi

```diff
@@ -0,0 +1,12 @@
+from typing_extensions import Literal
+
+PY_3: int
+
+def to_bytes(integer: int, length: int, byteorder: str) -> bytes: ...
+def from_bytes(bytesarray: bytes | bytearray, byteorder: str) -> int: ...
+
+class Counter:
+    byteorder: str
+    def __init__(self, nonce: bytes | bytearray, byteorder: Literal["big", "little"] = "big") -> None: ...
+    def __call__(self): ...
+    def reset(self) -> None: ...
```

Comment:
```suggestion
from typing_extensions import Literal
from typing import SupportsBytes
from _typeshed import ReadableBuffer

PY_3: bool

def to_bytes(integer: int, length: int, byteorder: Literal["big", "little"] ) -> bytes: ...
def from_bytes(bytesarray: SupportsBytes | ReadableBuffer, byteorder: Literal["big", "little"] ) -> int: ...

class Counter:
    byteorder: str
    def __init__(self, nonce: SupportsBytes | ReadableBuffer, byteorder: Literal["big", "little"] = "big") -> None: ...
    def __call__(self) -> bytes: ...
    def reset(self) -> None: ...
```
Mostly syncing the types with `int.to_bytes` and `int.from_bytes`, which these delegate to.